content/en/docs/setup/install/istioctl/index.md

```diff
@@ -48,6 +48,26 @@ security related configuration parameters:
 $ istioctl manifest apply --set values.global.mtls.enabled=true --set values.global.controlPlaneSecurityEnabled=true
 {{< /text >}}
 
+In general, you can use the `--set` flag in `istioctl` as you would with `Helm`. The major difference is you must
+prefix the `Helm` paths with `values.` because this is the path to the `Helm` pass-through API.
+
+## Install from external charts
+
+By default, `istioctl` uses compiled-in charts to generate the install manifest. These charts are released together with
+`istioctl` for auditing and customization purposes and can be found in the release tar in the
+`install/kubernetes/operator/charts` directory.
+`Istioctl` can also use external charts rather than the compiled-in ones. To select external charts, set
```

Comment:
```suggestion
`istioctl` can also use external charts rather than the compiled-in ones. To select external charts, set
```

Reply:
afaics capitalizing symbols is the norm in the docs.

Reply:
I don't think grammar rules apply to things in `code blocks`, but will let docs owners decide

Reply:
Other than in headings, we generally prefer to keep commands in lower case, even at the beginning of a sentence, which isn't ideal. The alternative is to change the sentence to:
```
The `istioctl` command can also use ...
```